internal/kibana/data_view/schema.go

```diff
@@ -334,8 +340,15 @@ func dataViewFromResponse(resp data_views.DataViewResponseObjectDataView) apiDat
 
 		if params, ok := formatMap["params"]; ok {
 			if paramsMap, ok := params.(map[string]interface{}); ok {
+				fieldFormatParams := apiFieldFormatParams{}
```

Comment:
You're never setting `apiFormat.Params = &fieldFormatParams` here and so the params are always nil (this is causing the current test failures).